cmd/compute-domain-daemon/podmanager.go

```diff
@@ -59,6 +60,7 @@ func NewPodManager(config *ManagerConfig, getComputeDomain GetComputeDomainFunc)
 		getComputeDomain: getComputeDomain,
 		podInformer:      podInformer,
 		informerFactory:  informerFactory,
+		mutationCache:    *mutationCache,
```

Comment:
Is there a reason to be making a copy of the mutation cache here instead of just tracking the pointer?